dojo/templates/dojo/view_finding.html

```diff
@@ -1590,6 +1601,54 @@ <h4>Credential
         });
     </script>
 
+    <script type="application/javascript">
```

Comment:
Duplication. Maybe we should move this into a separate file? Like some`crivo.js` where we dump the functions we use in multiple views.